build/evergreen_config_lib/tasks.py

```diff
@@ -920,17 +920,20 @@ def to_dict(self):
             func('fetch build', BUILD_NAME=self.depends_on['name']))
 
         stapling = 'mustStaple'
-        if self.test in [ 'test_3', 'test_4', 'soft_fail_test']:
+        if self.test in [ 'test_3', 'test_4', 'soft_fail_test', 'cache']:
             stapling = 'disableStapling'
         if self.test in [ 'malicious_server_test_1', 'malicious_server_test_2' ]:
             stapling = 'mustStaple-disableStapling'
 
         orchestration_file = '%s-basic-tls-ocsp-%s' % (self.cert, stapling)
         orchestration = bootstrap(TOPOLOGY='server', SSL='ssl', OCSP='on', ORCHESTRATION_FILE=orchestration_file)
 
-        commands.append(shell_mongoc('TEST_COLUMN=%s CERT_TYPE=%s USE_DELEGATE=%s sh .evergreen/run-ocsp-responder.sh' % (self.test.upper(), self.cert, 'on' if self.delegate == 'delegate' else 'off')))
+        commands.append(shell_mongoc('TEST_COLUMN=%s CERT_TYPE=%s USE_DELEGATE=%s sh .evergreen/run-ocsp-responder.sh' % ('TEST_4' if self.test == 'cache' else self.test.upper(), self.cert, 'on' if self.delegate == 'delegate' else 'off')))
```

Comment:
The cache test is a bit of a special case. I'd suggest adding an `if` to split up this line, or commenting saying we're using `TEST_4`'s responder configuration for the cache test.

Reply:
Added comment